src/dodal/devices/athena/panda/flyingpanda.py

```diff
@@ -0,0 +1,44 @@
+from typing import Dict, Iterator
+
+from bluesky.protocols import Descriptor, Flyable, PartialEvent
+from ophyd_async.core import AsyncStatus, wait_for_value
+from ophyd_async.panda import PandA
+
+import dodal.devices.athena.panda.tables as tables
+
+
+class FlyingPanda(Flyable):
```

Comment:
Should: I think some docstrings would be good on this. What's the device useful for?

Reply:
We've discussed and think FlyingPanda should be in ophyd-async alongside Panda as you suggest, so will resolve this and add change there.